infra/pipelines/pipeline.json

```diff
@@ -0,0 +1,764 @@
+{
+  "Repository": "corefx",
+  "Definitions": {
+    "Path": "..\\definitions",
+    "Type": "VSTS",
+    "BaseUrl":  "https://devdiv.visualstudio.com/DefaultCollection"
+  },
```

Comment:
Should `Definitions` be an array so it can point to multiple definition sources? If we share, say, the Package-Publish def, I'm wondering if this should be the injection point. I think changing this can wait until we have an actual plan for sharing the def, though.